EasyPost/Utilities/Rates.cs

```diff
@@ -64,12 +64,13 @@ public static Rate GetLowestObjectRate(List<Rate> rates, List<string>? includeCa
                 float rateValue = float.Parse(rate.rate);
                 float lowestRateValue = float.Parse(lowestRate.rate);
 
-                // if this rate is lower than the lowest rate, set it to this rate
-                if (rateValue < lowestRateValue)
+                // if this rate is higher than or the same as the lowest rate, skip it
+                if (lowestRateValue <= rateValue)
```

Comment:
Please do not alter this as it will break someones integration. With a particular carrier mix, customers have come to rely on the behavior of "last lowest rate gets returned". It's a terrible thing to rely on, but I promise you someone will right in about this if we alter it (even if this is a smart altering to do less OR equals).